### ISSUE TITLE
On publish, pop-up a dialog showing what users will be notified

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#193]
Also consider adding a checkbox to this dialog "[ ] Don't Send Notifications", crossing the icons out in the dialog when it's checked (to communicate that this wont happen).

**Comment by local-minimum** [comment]
Either I dont understand this or it dounds possiblu bad.

If a user can decide what notifications to get, that is good.

But allowing a user to decide if someone else gets notified soinds like a violation of yhe trust that you want in a notification system. It also allows for bad behavior in comments. So I dont see a clear use case that justifies these two negative aspects

**Comment by povrazor** [comment]
Okay, let me rephrase this.

When you use somebody's @name in a post or comment, that effectively means this post/comment will pop up in their feed. This is what I mean by a notification. The message is just a warning to the user that "hey, does this person really need to hear about this?". Newer users wont necessarily be aware of this behaviour, so it's a website education enhancement.

**Comment by local-minimum** [comment]
That is more fine, as long as it isn't annoying since every time you comnent on your own game you are likely to at someone.

**Comment by povrazor** [comment]
Yeah. I want a confirmation popup anyway (@'s or not), just the focus up until now has been function over usability.
